frontend/src/scss/PrimeStyles.scss

```diff
@@ -231,7 +231,7 @@
 .prime-modal-buttons button {
   margin-left: 1em;
 }
-.prime-modal-text {
+.sr-modal-content p {
```

Comment:
This style and `prime-modal-buttons` above can be moved to QueueItem.tsx, they're not used anywhere else.